tests/sprinkles/sprinkles-type-tests.ts

```diff
@@ -283,4 +283,10 @@ const noop = (...args: Array<any>) => {};
   > = ['row'];
 
   noop(invalidRequiredValue);
+
+  conditionalProperties.conditions.conditionNames.includes('mobile');
+  conditionalProperties.conditions.conditionNames.includes('tablet');
+  conditionalProperties.conditions.conditionNames.includes('desktop');
+  // @ts-expect-error
+  conditionalProperties.conditions.conditionNames.includes('missingCondition');
```

Comment:
Could we change this to validate the public API (e.g. `sprinkles.conditions`) instead?